src/runner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "fluvio-runner-local"
-version = "0.4.0"
+name = "fluvio-run"
+version = "0.1.0-alpha.1"
```

Comment:
let's remove `alpha` since we are not publishing and causing problem

Reply:
I think it would be good for us to use the Platform version for the `fluvio-run` version. I set it to 0.7.4 since that is the version this will be released in, let me know if you prefer something different.